youtube_dl/extractor/go.py

```diff
@@ -138,7 +138,7 @@ def _real_extract(self, url):
                     # from http://freeform.go.com/shows/shadowhunters/episodes/season-2/1-this-guilty-blood
                     r'data-video-id=["\']*(VDKA\w+)',
                     # https://abc.com/shows/the-rookie/episode-guide/season-02/03-the-bet
-                    r'\b(?:video)?id["\']\s*:\s*["\'](VDKA\w+)'
```

Comment:
Do not touch old patterns.

Reply:
Sorry for not minding my own business, but it looks like your requested change was implemented by @tmthywynn8 a couple of months ago.  I'm not that familiar with the GitHub workflow, but it looks like this is just waiting for your approval.

Reply:
Yes. I made the necessary changes in commit 783d296 and requested another review to make sure that I fulfilled the requirements. I'm not sure what the procedure is suppose to be, so maybe requesting another review was the wrong thing to do

Reply:
I'm not sure yet but I think removing the original pattern (by replacing it with the new pattern) fixed the problem, and putting it back has re-broken it.  My suspicion is that both patterns are matched on the page in question but one returns an invalid ID.

If I'm right, undoing the requested change will get it working again.  But that's not really a solution, and again I'm not sure yet.

Reply:
Sorry it took me so long to get to this, but I just tested #25216 and it works for me as-is.  So my theory above is completely wrong.